CodeEdit/Features/AppPreferences/Model/Terminal/TerminalPreferences.swift

```diff
@@ -37,6 +40,8 @@ extension AppPreferences {
             self.optionAsMeta = try container.decodeIfPresent(Bool.self, forKey: .optionAsMeta) ?? false
             self.shell = try container.decodeIfPresent(TerminalShell.self, forKey: .shell) ?? .system
             self.font = try container.decodeIfPresent(TerminalFont.self, forKey: .font) ?? .init()
+            self.cursorStyle = try container.decodeIfPresent(TerminalCursorStyle.self,
+                                                             forKey: .cursorStyle) ?? .steadyBlock
```

Comment:
This needs to be formatted this way (see [code style](https://github.com/CodeEditApp/CodeEdit/wiki/Code-Style#parameter-alignment))

```suggestion
            self.cursorStyle = try container.decodeIfPresent(
                TerminalCursorStyle.self,
                forKey: .cursorStyle
            ) ?? .steadyBlock
```